app/controllers/result_filters/qualification_controller.rb

```diff
@@ -0,0 +1,24 @@
+module ResultFilters
+  class QualificationController < ApplicationController
+    include FilterParameters
+
+    before_action :create_view
+
+    def new; end
+
+    def create
+      if @view.qualification_selected?
+        redirect_to results_path(filter_params)
+      else
+        flash[:error] = "You must make at least one selection"
```

Comment:
thats not the current behaviour

Reply:
I've asked @jesseyuen to review on the card. I'll update as required.

Reply:
Update from Jesse. This is the correct behaviour.